tst/main.cpp

```diff
@@ -75,6 +75,16 @@ int main(int argc, char** argv)
         if (trial >= MAX_TRIALS) {
             ::testing::GTEST_FLAG(break_on_failure) = breakOnFailure;
         }
+
+        // Increase logging verbosity on retry
+        if (trial >= 1) {
```

Comment:
Since there is a possibility it retries and succeeds, can consider changing this to `if (trial >= MAX_TRIALS)` so it's only verbose on the last attempt.

Reply:
That's a good suggestion.. I think maybe even better would be to buffer the logs and capture all failures.. because then the first failure won't have the verbose logs.

Reply:
Can do in a follow -up